localisation/00_zuwc_united_localisation.csv

```diff
@@ -2885,6 +2737,109 @@ EVTOPT_MNM_4018;Cognition is what I live for.;;Erkenntnis ist das, wof
+EVTDESC_WCFW_60;It seems that despite all the efforts your clan has made to leave the Horde, they have all failed. Perhaps all your decisions were doomed to failure and the Frostwolves are destined to stay with the Horde after all? It is difficult to say and, unfortunately, it only brings more questions than answers. You hope that perhaps all is not lost and that it is only a matter of rebuilding. Only time will tell.;Il semble que malgr� tous les efforts de votre clan pour quitter la Horde, ils ont tous �chou�. Peut-�tre que toutes vos d�cisions �taient vou�es � l'�chec et que les Frostwolves sont destin�s � rester avec la Horde apr�s tout? C'est difficile � dire et, malheureusement, cela n'apporte que plus de questions que de r�ponses. Vous esp�rez que tout n'est peut-�tre pas perdu et qu'il ne s'agit que de reconstruire. Seul l'avenir le dira.;Es scheint, dass trotz all der Bem�hungen, die Euer Clan unternommen hat, um die Horde zu verlassen, alle vergebens war. Vielleicht sind all Eure Entscheidungen zum Scheitern verurteilt und die Frostw�lfe sind doch dazu bestimmt, bei der Horde zu bleiben? Es ist schwer zu sagen und bringt leider nur mehr Fragen als Antworten. Ihr hofft, dass vielleicht nicht alles verloren ist und dass es nur eine Sache des Wiederaufbaus ist. Nur die Zeit wird es zeigen.;;;;;;;;;;;x
+EVTOPT_WCFW_60;We are Frostwolves, we will endure.;Nous sommes des Loup-de-givre, nous endurerons.;Wir sind Frostw�lfe, wir werden es ertragen.;;;;;;;;;;;x
+EVTTITLE_WCFW_61;Frostwolf Refusal!;Refus du Loup-de-givre!;Ablehnung der Frostw�lfe!;;;;;;;;;;;x
+EVTDESC_WCFW_61;Despite the fairly clear orders you gave to [From.GetFirstName], the Frostwolf Clan refused to leave the Horde. Filled with rage at their disobedience, you gather a company of your best and strongest warriors. With them and their wolves, you ride to put an end to the Frostwolf filth that has been tormenting the Horde for far too long. You intend to strike at night and make sure no Frostwolf will walk away from this.;Malgr� les ordres assez clairs que vous aviez donn�s � [From.GetFirstName], le clan Loup-de-givre a refus� de quitter la Horde. Rempli de rage face � leur d�sob�issance, vous rassemblez une compagnie de vos meilleurs et plus forts guerriers. Avec eux et leurs loups, vous montez � cheval pour mettre fin � cette salet� de Loup-de-givre qui tourmente la Horde depuis bien trop longtemps. Vous avez l'intention de frapper la nuit et de vous assurer qu'aucun Loup-de-givre ne s'en sortira.;Trotz der eindeutigen Befehle, die Ihr [From.GetFirstName] gegeben habt, weigerte sich der Frostwolf-Clan, die Horde zu verlassen. Voller Wut �ber ihren Ungehorsam versammelt Ihr eine Gruppe Eurer besten und st�rksten Krieger. Mit ihnen und ihren W�lfen reitet Ihr, um dem Frostwolf-Abschaum, der die Horde viel zu lange gequ�lt hat, ein Ende zu setzen. Ihr beabsichtigt, nachts zuzuschlagen und sicherzustellen, dass kein Frostwolf entkommt.;;;;;;;;;;;x
```

Comment:
> Despite the fairly clear orders you gave to [From.GetFirstName]

`[From.GetFirstName]` don't work here, because `From`it's province

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/69394519-47d2c800-0ced-11ea-9b44-fa2c40c2babc.png)

</details>